editor/editor_properties_array_dict.h

```diff
@@ -105,6 +105,7 @@ class EditorPropertyArray : public EditorProperty {
 	void _change_type_menu(int p_index);
 
 	void _object_id_selected(const String &p_property, ObjectID p_id);
+	void _remove_pressed(int index);
```

Comment:
Should be `p_index` (same in *Dictionary method).

Reply:
Updated array\dictionary methods to p_index. I realise now Godot uses p to indicate parameter, sorry about that.